src/wallet/wallet2.cpp

```diff
@@ -788,7 +788,7 @@ void wallet2::scan_output(const cryptonote::account_keys &keys, const cryptonote
   {
     tx_scan_info.money_transfered = tools::decodeRct(tx.rct_signatures, tx_scan_info.received->derivation, i, tx_scan_info.mask);
   }
-  tx_money_got_in_outs[tx_scan_info.received->index] = tx_scan_info.money_transfered;
+  tx_money_got_in_outs[tx_scan_info.received->index] += tx_scan_info.money_transfered;
```

Comment:
That seems rather important. Is it possible to get a test to prevent a regression?

Reply:
Sounds like something that might fit the functional tests once they're revived.